src/app/jobs/weatherevents.job.ts

```diff
@@ -0,0 +1,55 @@
+import { Job } from '../../common/job';
+import { IContainer } from '../../common/types';
+import { TextChannel } from 'discord.js';
+import ms from 'ms';
+
+interface IWeather_Event {
+  features?: (IFeaturesEntity)[] | null;
+}
+interface IFeaturesEntity {
+  properties: IProperties;
+}
+
+interface IProperties {
+  severity: string;
+  headline: string;
+  senderName: string;
+}
+
+export class WeatherEventsJob extends Job {
+  public name: string = 'Weather Events';
+  public interval: number =  ms('1h');
+
+  private _ENDPOINT: string = 'https://api.weather.gov/alerts/active?area=FL';
+
+  constructor() {
+    super();
+  }
+
+  public async execute(container: IContainer) {
+    const channel = (container.guildService.getChannel('weather_events')) as TextChannel;
+    const resp = await container.httpService.get(this._ENDPOINT);

```

Comment:
Arent we stepping away from using `httpService`? I might be wrong on this.

Reply:
In this context either way it doesn't matter. My PR needs to be split down before any changes can be enforced.

Reply:
In the event that that change can be enforced, someone can comment on this PR or open an issue about it and I'll change it back to what it needs to be.